azdev/config/ext_pylintrc

```diff
@@ -1,17 +1,14 @@
 [MASTER]
+ignore=tests,generated,vendored_sdks,privates
 ignore-patterns=test_*
```

Comment:
I think we can remove this line as it is simply wrong (https://github.com/Azure/azure-cli/pull/18003#discussion_r628096102).

```suggestion
```